google-cloud-bigquery/lib/google/cloud/bigquery/dataset.rb

```diff
@@ -1380,9 +1380,9 @@ def load_job table_id, file, format: nil, create: nil, write: nil,
         # Request](https://cloud.google.com/bigquery/loading-data-post-request#multipart).
         #
         # @param [String] table_id The destination table to load the data into.
-        # @param [File, Google::Cloud::Storage::File, String, URI] file A file
-        #   or the URI of a Google Cloud Storage file containing data to load
-        #   into the table.
+        # @param [File, Google::Cloud::Storage::File, String, URI, Array] files
```

Comment:
@quartzmo Can you offer some suggestions on how to clarify that the array must be fore remote file URLs and not a local file?

Reply:
I think I'd go with something like this:

```ruby
        # @param [File, Google::Cloud::Storage::File, String, URI, 
        #   Array<Google::Cloud::Storage::File>, Array<String|URI>] files
        #   A file or files containing data to load into the table.
        #   This may be a file or Google Cloud Storage file object; a string or 
        #   URI referencing a Google Cloud Storage file; an array of Google 
        #   Cloud Storage file objects; or an array of strings or URIs referencing 
        #   Google Cloud Storage files.
```

It's usually better in these cases not to be too economical with words, but to really spec it out.

Reply:
Could you use `Array<Google::Cloud::Storage::File|String|URI>` instead of `Array<Google::Cloud::Storage::File>, Array<String|URI>`?

Reply:
The latter seems more correct, but I don't know anything about how documentation is produced. Let me know which will work (or how I test that) and I'll make the update.

Reply:
Run `rake ci`, then open `doc/index.html` in a browser and take a look at your additions. The `Google::Cloud::Storage::File` inside `Array<Google::Cloud::Storage::File>` should be a link to the `File` docs.

Reply:
@blowmage Hmmm, I think I should have suggested: `Array<Google::Cloud::Storage::File>, Array<String, URI>`, which might be the same as Array<(Google::Cloud::Storage::File|(String, URI))>, although I'm not sure how far you can push this notation. Anyway, the point is: Either an array of files objects, OR an array of string AND/OR  URI objects. (Correct?)

Reply:
I don’t see why you couldn’t mix all 3 types that represent a GCS URL.

Reply:
Programmatically, you can mix all three types. Only a local `File` is different. I suppose it's up to you what you want to expose in the API (and be committed to).

I ran `rake ci` and looked at `docs/index.html` (thanks for the help) but yard doesn't link it either way and also didn't have `Google::Cloud::Storage::File` linked with the syntax that was there before.

[Yard docs](https://yardoc.org/types.html#see_results) show an "array of various objects" as this:

```
Array<String, Symbol, #read>
```

So I went with comma-separated.